37_setting_up_production_like_kubernetes_cluster_part_3_13_dec_2022/references.md

```diff
@@ -0,0 +1,13 @@
+## References on: Setting up a production-like Kubernetes cluster for the first time, part 4, 13 Dec 2022
```

Comment:
please remove separate files references.md, please include the contents into README.md